action.php

```diff
@@ -48,8 +48,10 @@ function actionrenderer(Doku_Event $event, $param) {
 			$returnonly = $args[3] ?: false;
 			$linktype = $args[4] ?: 'content';
 
+			resolve_pageid(getNS($ID), $id, $exists);
+
 			global $ID;
-			if (!$this->m_helper->isExcluded($ID) && page_exists($id) && $id != $ID) {
+			if (!$this->m_helper->isExcluded($ID) && $exists && $id != $ID) {
```

Comment:
What does this change accomplish?

Reply:
If there're relative links, autotooltip did not recognize them as relative link, but absolute link.

Example:
In "namespace:some_page", there're link like "[[other_page]]".
Since this is relative link, this link should be resolved as ":namespace:other_page".
However autotooltip resolved as ":other_page" and consider linked page does not exist.

So I re-positioned link resolve logic to early stage.